### PR TITLE
feature/gsk-1692-catalog-not-showing-full-list-of-tests

### PR DESCRIPTION
**Pull request by andreybavt**

### CODE REVIEW

File: frontend/src/stores/catalog.ts [hunk 1]
```diff
@@ -18,7 +18,7 @@ interface State {
 
 function latestVersions<E extends CallableDTO>(data?: Array<E>): Array<E> {
     return chain(data ?? [])
-      .groupBy(func => func.module ?? `${func.module}.${func.name}`)
+      .groupBy(func => `${func.module}.${func.name}` ?? func.module)
```
Comment (kevinmessiaen) [inline frontend/src/stores/catalog.ts:21]: Actually this is the same as `${func.module}.${func.name}` since the `??` would never be triggered.

**Review by kevinmessiaen — COMMENTED** [review]
IMO we should probably have something that check the `displayName` if set to have something more robust. As the current solution will merge all lambda generated from same package (I know @Googleton  faced the issue in the push feature) and also if we rename a module (happened during a version migration)

It's similar to this PR: https://github.com/Giskard-AI/giskard/pull/1348/files

File: frontend/src/stores/catalog.ts (hunk 1 above)
Comment (andreybavt) [reply frontend/src/stores/catalog.ts:21]: ah yes, sure 🤦 , let me clean it up

Comment (andreybavt) [reply frontend/src/stores/catalog.ts:21]: I ported a similar fix to https://github.com/Giskard-AI/giskard/pull/1348

https://github.com/Giskard-AI/giskard/pull/1348/commits/dec4a5ad8d9df11c1fe12f5a0936bf1ba51620c2

**Comment by andreybavt** [comment]
superseded by https://github.com/Giskard-AI/giskard/pull/1348

**Pull request closed without merging**